console/src/components/UIKit/atoms/Spinner/Spinner.ts

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import {
+  color,
+  border,
+  typography,
+  layout,
+  space,
+  shadow,
+  ColorProps,
+  BorderProps,
+  TypographyProps,
+  LayoutProps,
+  SpaceProps,
+  ShadowProps,
+} from 'styled-system';
+
+interface BoxProps extends Omit<React.ComponentPropsWithRef<'div'>, 'color'> {}
+const Box = ('div' as any) as React.FC<BoxProps>;
```

Comment:
Can you move it to a separate file, for example in `console/src/components/UIKit/atoms/Box/index.ts` as it will be needed in other places as well?

Reply:
Right. I fix it.

https://github.com/hasura/graphql-engine/pull/4504/commits/6cba41ec5aa452121e7cb1d75e8e1b23533cbafc